### ISSUE TITLE
Not generating anything

### ISSUE THREAD

**Issue by tadkaleb** [ItsBlackGear/CavesAndCliffsBackport#35]
Im using this mod in the Better Minecraft modpaack and none of the cave generations are spawning. Are they biome specific?

**Comment by ItsBlackGear** [comment]
It could also be caused by other terrain generators, if you're using
terraforged as an example, they will not generate

Make sure that they are generating by creating a new world and using
/locatebiome cavesandcliffs:dripstone_caves
But it has to be underground of below y68

**Comment by devjavu7** [comment]
Using Terraforged, the cave biomes will now generate.

**Comment by Jukabofy** [comment]
That's a known "issue", instead of cave biomes Terraforged and OTG generate mountain biomes

**Comment by ItsBlackGear** [comment]
yup, some terrain generators as Terraforged or OTG will prevent the generation of cave biomes

**Comment by devjavu7** [comment]
Yes I know, but I am telling you, I can locate the cave biomes with the commands and there is actually a biome in that place when using Terraforged. I have no idea why it works now, all that I know is that it does. At least in 1.16
